avro/src/main/scala/magnolify/avro/AvroType.scala

```diff
@@ -89,9 +88,9 @@ object AvroField {
         false,
         caseClass.parameters.map { p =>
           new Schema.Field(
-            cm map p.label,
+            cm.map(p.label),
```

Comment:
I avoid infix for most cases, just for the sake of consistency. The only exceptions maybe `zip/union/++` where both sides are equivalent.